### DISCUSSION TITLE
How to use Template feature

### DISCUSSION THREAD

**Discussion by gunir** [filips123/PWAsForFirefox#389] [Q&A] [answered]
I just know that it's possible to use template, but how because I couldn't find a guide in the wiki ?

Post: https://github.com/filips123/PWAsForFirefox/issues/386#issuecomment-1686720851

**Comment by filips123** [comment, marked as answer]
1. Create an empty folder that will be used as a profile template.
2. Create `user.js` file that configures your preferences. For example:

```js
user_pref("fission.autostart", false);
user_pref("dom.ipc.processCount", 1);
```

3. In the extension settings, set the default template to a path to your template directory.
4. All new profiles will contain your `user.js` file with configured preferences.

This will only affect new profiles. If you want to quickly disable Fission in existing ones, just copy `user.js` to all your profile directories.

**Reply by gunir** [reply]
Thanks very much!
